src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -15,7 +15,6 @@
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
 
     <ProjectDirectory>$([MSBuild]::NormalizeDirectory('$(SrcDir)', '$(RepositoryName)'))</ProjectDirectory>
-    <PackagesOutput>$([MSBuild]::NormalizeDirectory('$(ProjectDirectory)', 'artifacts', 'packages', '$(Configuration)', 'NonShipping'))</PackagesOutput>
```

Comment:
I have no idea why this only ever considered non shipping assets.

Reply:
Conjecture, when intermediates where in play, it was the only package produced in the repo's outer build and was NonShipping

Reply:
That's a good guess.